tests/Unit/OAuth/Infrastructure/Provider/StatefulTwitterProviderDouble.php

```
@@ -0,0 +1,31 @@
+final class StatefulTwitterProviderDouble extends Twitter
+{
+    public function __construct()
+    {
+    }
+
+    /**
+     * @param string $grant
+     * @param array<string, string> $options
+     */
+    #[\Override]
+    public function getAccessToken($grant, array $options = []): AccessToken
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$grant'. (undefined)

(UnusedFormalParameter)